### DISCUSSION TITLE
Production & Version Controll

### DISCUSSION THREAD

**Discussion by GeorgeFourkas** [MohmmedAshraf/laravel-translations#59] [Q&A]
Hello! 
First of all, congratulations for this package. It's a good idea. 
I have a question regarding version control of the Laravel app, and the usage of this package. 

How do you manage the version control of the translations, on production? I mean, wouldn't it have conflicts with laravel development enviroment?

For example, suppose the following scenario for a blog. We have the dev env, where we finish the development, we push to app to the production servers, and after 5 days we make a change to the translation files through the UI and we publish the translations. when we make a change to the functionality of the app, won't it get conflicted between the 2 environments? Since we would have uncomitted changes to the language files on production?

Does the package offer a solution to that?

Thank you for your time, and for developing this package :)

**Comment by MohmmedAshraf** [comment]
Hey @GeorgeFourkas, it makes sense. Perhaps we can include an option in the translation publishing dialog to download the language directory instead of publishing it directly to production. This would replace the local directory. What are your thoughts on this?
